samples/msal-node-samples/on-behalf-of-distributed-cache/src/AxiosHelper.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import axios from "axios";
+
+class AxiosHelper {
+
+    /**
+     * Makes an Authorization "Bearer" request with the given accessToken to the given endpoint.
+     * @param endpoint
+     * @param accessToken
```

Comment:
params?: Record<string, string> is not defined and it's really unclear what it a does.

Reply:
added a comment for the argument -it's an Axios request parameter object, which apparently has the type `any` in Axios. Not sure why this is so though, in practice it will be always be a JS object (hence key-value pairs) as far as I can see